src/models/Movie/getById.ts

```diff
@@ -0,0 +1,23 @@
+import { parseMovieModel } from './parseMovieData';
+import { cdaClient } from '../../utils/contentful';
+import { Movie, MOVIE_CONTENT_TYPE } from './type';
+import { CONTENTFUL_INCLUDE, ContentfulIncludeOptions } from '../../types/contentful';
+
+type QueryParamOptions = {
+	include?: ContentfulIncludeOptions;
+};
+
+export default async function getById(
+	id: string,
+	{ include = CONTENTFUL_INCLUDE.noInclude }: QueryParamOptions = {}
+): Promise<Movie | null> {
+	const response = await cdaClient.getEntry(id, { include });
+
+	if (response?.sys?.contentType?.sys?.id !== MOVIE_CONTENT_TYPE) {
```

Comment:
Could we extract `response?.sys?.contentType?.sys?.id` into a variable since it is not an immediately obvious comparison in the if's conditional section.

---
So we could have
```
const movieTypeFromContentful = response?.sys?.contentType?.sys?.id;

if( movieTypeFromContentful == MOVIE_CONTENT_TYPE )
```

Reply:
I think a comment here would be enough instead of creating a variable:
```suggestion
        // Contentful getEntry can retrieve any content type, need to check
	if (response?.sys?.contentType?.sys?.id !== MOVIE_CONTENT_TYPE) {
```